onboarding_documentation/ingestion/ingestion_concept.md

```diff
@@ -0,0 +1,90 @@
+
+## Where are ingested (meta)data stored?
+
+Metadata are stored on CPG’s database, [Metamist](https://sample-metadata.populationgenomics.org.au/). Data are stored in [Google Cloud Platform](https://console.cloud.google.com/storage/browser?hl=en-AU&project=cpg-sandbox&prefix=&forceOnBucketsSortingFiltering=true) ‘projects’.
+
+## What happens after (meta)data are ingested?
+
+Ingested (meta)data are accessible using CPG data storage and in the production instance of Metamist. CPG analysts can now access these programmatically to use as input for their analyses using CPG pipelines. Further transformations may also be applied to the original meta(data) through CPG pipeline stages such as Quality Control (QC). The creation of new records in Metamist captures provenance metadata and usage metadata are captured on an ongoing basis to provide a record of how data are used in CPG workflows. Metadata, once uploaded, are not necessarily static and can be updated as required to include newly acquired metadata, for example.
+
+## How are projects initiated?
+
+The catalyst for projects is all context-specific and there is no single specific answer to this question. Projects are supported by the Project Management team, which maintains an [AirTable](https://airtable.com/app62isJvsSz0ziWT/tblFpqx87JOvUlSF6/viwAAW4LjI1hqzIvS?blocks=bipyLQRZiDX3CWVzM) database to track projects. This is an excellent resource that can be used to understand details surrounding a project’s conception, and may include links to external explanatory documents. Background information on a project can help us to understand the importance of the (meta)data that we are working with and their intended use after ingestion has occurred.
+
+‘Projects’ can refer to work where the CPG is collaborating with external organisations to process their genomic data and it can also refer to analysis projects undertaken by the CPG internally.
+
+## How are participants recruited and sequenced for projects?
+
+Depending on the project, participants may be recruited specifically for the study, or may have been recruited into a previous study and consented to reanalysis in future studies. A lot of projects are a mix of old and new data. The scale of the recruitment process varies between small (a few dozen participants, over the course of weeks or months) and large (thousands of participants, over the course of years), depending on the funding and research priority.
+
+## Why do researchers or project leads collaborate with the CPG to store and/or process their (meta)data?
+
+CPG provides genomic storage, processing, analysis and products to collaborators at no additional cost (beyond operating costs). This can allow groups of researchers to access advanced genomic analyses and tools that they would otherwise have access to with the resources they have at hand. The CPG generally does not operate in a for-profit way.
+
+## How does the CPG acquire (meta)data for ingestion?
```

Comment:
I appreciate the detail in this section re some of the issues with common modes of data / metadata transfer